### PR TITLE
feat: Analytics for choosing to stick the menu - MEED-1841 - Meeds-io/meeds#776

### PR DESCRIPTION
**Pull request by SaraBoutej**
Prior to this change, no analytics for stick and unstick the left menu are added, after this change, added analytics for stick/ unstick operation

### CODE REVIEW

**Review by boubaker — DISMISSED** [review]

File: analytics-webapps/src/main/webapp/js/statistic-collection.js [hunk 1]
```diff
@@ -147,13 +147,13 @@ function() {
...
     },
-    addStatisticMenuStickiness: function (eventDetail) {
+    addStatisticMenuStickiness: function (menuSticked) {
       const spaceLeftNavigationStickiness = {
         'module': 'portal',
         'subModule': 'ui',
         'userId': eXo.env.portal.userIdentityId,
         'userName': eXo.env.portal.userName,
-        'operation': eventDetail && 'sticky' || 'unsticky',
+        'operation': menuSticked && 'stick' || 'unstick',
```
Comment (boubaker) [inline analytics-webapps/src/main/webapp/js/statistic-collection.js:156]: We should change those labels as well :

https://github.com/Meeds-io/analytics/commit/821cf31a4b23c54d2c16f9855a294ca221abb9ee#diff-c84780a7dc63638d1d2c361fc56b6c108c5cf5761c11c40688abccb27e41294cR397-R398

Comment (SaraBoutej) [reply analytics-webapps/src/main/webapp/js/statistic-collection.js:156]: changed!

Comment (boubaker) [reply analytics-webapps/src/main/webapp/js/statistic-collection.js:156]: Same for all I18N files.
In fact, it's already synced with Crowdin, so you should delete all I18N keys like here:
https://github.com/Meeds-io/analytics/blob/236419507f389c54fb772bdaa6bd087376a498b1/analytics-webapps/src/main/resources/locale/portlet/Analytics_fr.properties#L397-L398
To make it regenerated again.

Comment (SaraBoutej) [reply analytics-webapps/src/main/webapp/js/statistic-collection.js:156]: done cc @boubaker

Comment (boubaker) [reply analytics-webapps/src/main/webapp/js/statistic-collection.js:156]: cool, thanks

**Review by boubaker — APPROVED** [review]

**Pull request merged**